pokedex/src/components/App.js

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+import logoPoke from '../assets/pokeball-logo.svg';
+import Filter from './filter/index';
+import SearchBar from './searchBar/index';
+import ContainMainCards from './containMinCards';
+import styles from './styles.module.scss';
+
+function App() {
+  const [pokemon, setPokemon] = useState({ info: [], isOpen: false });
+  const [stateFilter, setStateFilter] = useState(false);
+  const [tablePoke, setTablePoke] = useState({ info: [] });
+  const [form, setForm] = useState({ namePokemon: '' });
+
+  const handleChange = (e) => {
+    setForm((lastForm) => ({ ...lastForm, namePokemon: e.target.value }));
+    search(e.target.value);
+  };
+
+  const getPokemon = async (pokemon) => {
+    const res = await axios(`https://pokeapi.co/api/v2/pokemon/${pokemon}`);
+    setPokemon((last) => ({ ...last, info: [...last.info, res.data] }));
+    setTablePoke((last) => ({ ...last, info: [...last.info, res.data] }));
+  };
+
+  const search = (busquedaPokemon) => {
```

Comment:
Podríamos cambiar el nombre de esta función a pokemonFilter y el parámetro ponerlo en inglés para que tenga más lógica con toda la app 😄 

```suggestion
  const pokemonFilter = (pokemon) => {
```

Reply:
listo